### DISCUSSION TITLE
media-loading-indicator display

### DISCUSSION THREAD

**Discussion by EssLi** [muxinc/media-chrome#1009] [Q&A] [answered]
when the mouse move leave the video region, the media-loading-indicator disappear unless mouse move enter again。
i wanna media-loading-indicator display before the video frame loaded no matter where the mouse cursor is。
could you give some advice？

**Comment by luwes** [comment, marked as answer]
hi @EssLi, this is possible with a `noautohide` attribute on the `media-loading-indicator` element.
See here for an example https://github.com/muxinc/player.style/blob/main/themes/minimal/template.html#L265

**Comment by EssLi** [comment]
Thank you very much for your advice! I will give it a try.
